.travis.yml

```diff
@@ -0,0 +1,27 @@
+before_script:
+  - ruby --version
+
+env:
+  global:
+    - APM_TEST_PACKAGES="language-haml linter"
```

Comment:
I'm not following exactly what needs to be done here.  I'm assuming that atom-package-deps should be able to install the package instead of using the `APM_TEST_PACKAGES` environment variable.

So I took another look at atom-package-deps and its [readme](https://github.com/steelbrain/package-deps/blob/a920bfba1080c915223c3f5ee05261b4373d64fe/README.md) example shows specifying the value `["linter"]` to the key of `package-deps` in `package.json` and passing `linter-ruby` to the call to `install`.  It doesn't really make sense to be specifying dependencies in two different places, but I decided to try it anyway.  I removed setting the `APM_TEST_PACKAGES` environment variable, removed `langage-haml` from the `package-deps` array and passed `'language-haml'` to the call to `install`.  [That didn't work.](https://travis-ci.org/jschroeder9000/linter-haml/jobs/115610552)

So then I thought maybe I shouldn't have removed `language-haml` from the `package-deps` array.  I added it back and tried again.  [That didn't work, either.](https://travis-ci.org/jschroeder9000/linter-haml/jobs/115611056)

So I re-read the readme again and noticed it says "replace the example argument 'linter-ruby' with the name of _this_ Atom package".  So I thought maybe it's supposed to be the name of the package that's using it for some reason.  I left the `package-deps` array to contain both `linter` and `language-haml` and tried passing `linter-haml` to the call to `install`.  [That didn't work, either.](https://travis-ci.org/jschroeder9000/linter-haml/jobs/115611621)

Am I missing something?